PWGJE/EMCALJetTasks/AliEmcalJetUtilityConstSubtractor.h

```diff
@@ -49,6 +51,8 @@ class AliEmcalJetUtilityConstSubtractor : public AliEmcalJetUtility
   TClonesArray          *fParticlesSub;                       //!subtracted particle collection
   AliRhoParameter       *fRhoParam;                           //!event rho
   AliRhoParameter       *fRhomParam;                          //!event rhom
+  Double_t              fAlpha;                              //!pT weight exponent applied in const sub
```

Comment:
Hello,
are you sure that you want these fields as transient members (//!) ?
This means that the content of these fields won't be kept when an instance of this class is stored in a root file (which is necessary when the task is used on the grid).
If you change them into non-transient members (i.e. remove the ! after //) remember to increase the class version (last line of this file, ClassDef).
Best,
Salvatore